content/tutorials/planetary-projects/_index.md

```diff
@@ -65,6 +67,14 @@ Votes are not equal, but instead depend on the voting company's influence, which
 
 For example, a company employing 200 pioneers, 100 settlers, and 200 technicians has an influence of 200 * 1 + 100 * 1.25 + 200 * 1.5 = 625.
 
+#### Upkeep Costs
+
+The CoGC's programs require payment of an upkeep cost in the form of materials to go into effect.  
+
+![Program upkeep](cogc-upkeep.png)
+
+The materials required as upkeep are DW, MCG, PE and RAT.  The total amount required of each material in upkeep is determined by the number of player bases on the planet multiplied by the material multiplier.  The multipliers are 3x for DW, 4x for MCG and PE, and 2x for RAT.  If a planet has a single base the CoGC upkeep will be twice the multiplier values.
```

Comment:
The minimum upkeep is five times the multiplier values.

Reply:
Got 2x from Mjeno.  Updated text to say 5x.